lib/controllers/ServiceFabrikAdminController.js

```diff
@@ -623,6 +601,75 @@ class ServiceFabrikAdminController extends BaseController {
         .send({}));
   }
 
+  startMaintenance(req, res) {
+    logger.info('req.body=>', req.body);
+    return maintenanceManager
+      .getMaintenaceInfo()
+      .then(body => {
+        if (body === null) {
+          return maintenanceManager
+            .startMaintenace(req.body, req.user)
+            .then(body => res
+              .status(201)
+              .send(body));
+        } else {
+          res
+            .status(403)
+            .send({
+              errorMessage: 'System already in maintenance mode',
```

Comment:
Better to define error messages in some constants.

Reply:
As discussed currently we do not yet have a defined way of defining all messages in a single central location. Just changing it for this feature would lead it to be a bit confusing. Leaving this as-is as agreed.